src/core/image/image.controller.ts

```diff
@@ -0,0 +1,12 @@
+import type { FastifyInstance } from 'fastify';
+import { createResizeHandler } from './resize/resize.handler';
+import { createMetadataHandler } from './resize/metadata.handler';
+import { createCropHandler } from './resize/crop.handler';
+import { createThumbnailHandler } from './resize/thumbnail.handler';
+
+export const imageController = (fastify: FastifyInstance) => {
+  createMetadataHandler('/metadata', fastify);
```

Comment:
Dla mnie te handlery to powodują dodatkowy szum, nie wiem czy nie jest to przerost formy nad treścią. Zwłaszcza że objętościowo nie są jakoś strasznie wielkie.

Reply:
dla mnie wydaje się teraz trochę czytelniejsze głównie z powodu ilości typów (otypowanie route'a + json schema do walidacji). Ale tak poza tym to w sumie bez różnicy, mogę wrzucić wszystko do jednego pliku